torch/distributed/algorithms/quantization/quantization.py

```diff
@@ -10,7 +10,8 @@
 TORCH_HALF_MAX = torch.finfo(torch.float16).max
 
 class DQuantType(Enum):
-    FP16 = "fp16"
+    FP16 = "fp16",
```

Comment:
nit, and before this PR - it looks like `DQuantType` will be a public class used to pass in the `qtype` into auto_quantize API, should we add some documentation for it (can be done separately)?